app/components/ExternalLinksBlock/ExternalLinksBlock.jsx

```diff
@@ -7,7 +7,7 @@ import { zooTheme } from '../../theme';
 
 export const StyledExternalLinksBlock = styled.div`
   background-color: ${({ isItAProject }) => {
-    return (isItAProject) ? zooTheme.colors.background : 'white';
+    return (isItAProject) ? '#eff2f5' : 'white';
```

Comment:
Should this PR change `zooTheme.colors.background` to `zooTheme.colors.background.default`?

Reply:
`zooTheme.colors.background.default` doesn't exist, but I can do `zooTheme.colors.background.default`

Reply:
Yes, please do